### PR TITLE
Link to The Catalyst Project site

### PR DESCRIPTION
**Pull request by NPDebs**
This PR addresses issue #771.

**Changes made:**
- Link to the Catalyst website from navbar and open-incubator page.

**Live [deploy preview](https://deploy-preview-772--ols-bebatut.netlify.app/open-incubator#catalyst) here...**

**To be done:**
- [x] Add the Catalyst logo based on the outcome of [this discussion](https://github.com/open-life-science/open-life-science.github.io/issues/771#issuecomment-2037520884).

Thanks for reviewing!

### CODE REVIEW

**Comment by NPDebs** [comment]
Side note - I had alternatively intended to use [this icon](https://thenounproject.com/icon/friendship-money-1258376/) for Fiscal Sponsprship but they look to me like robbers. 😄

**Comment by NPDebs** [comment]
So, I had an idea. Rather than having just black icons, I thought to use our OLS green (`#48b87d`)...as seen below:

Screenshot (702)

But, I am advising myself against it for three (3) major reasons:
1. I think the green is much too light.
2. The colour doesn't seat well next to the Catalyst icon.
3. Every other icon (e.g three pillars) is black.

I will revert the latest commit, depending on what you both think. 🙏

**Comment by yochannah** [comment]
I agree - it looks better in black, best to revert!

File: open-incubator.md [hunk 1]
```diff
@@ -22,26 +26,45 @@ streams:
     title: Fellowship Programmes
     description: |
       Our flagship Fellowship is the Resident Fellow programmes.
```
Comment (yochannah) [inline open-incubator.md:28]:
```suggestion
      Our flagship incubator service is the Resident Fellow programme, where individuals work with OLS to further OLS's goals whilst exploring opportunities for personal growth.
```

File: images/icons/noun-writing-6279702.svg
Comment (yochannah) [inline images/icons/noun-writing-6279702.svg:1]: okay, super picky comment here - sorry! 

This file has much thinner lines than all the others. I'd suggest we either find a different icon or modify the icon so its lines match the weight of the others. Visal example here: 
image

tagging @jilaga as she may be able to tweak the image? @NPDebs I'll leave it up to you to decide if that's better or if a different image is easier.

Comment (yochannah) [reply images/icons/noun-writing-6279702.svg:1]: looking at the svg file... you MIGHT be able to fake it by giving the image a small stroke.

File: open-incubator.md (hunk 1 above)
Comment (NPDebs) [reply open-incubator.md]: Beautifully reworded. Thanks, Yo!

File: images/icons/noun-writing-6279702.svg
Comment (NPDebs) [reply images/icons/noun-writing-6279702.svg:1]: Picky eyes are useful too. :)

So...the stroke idea proved fruitful. I gave it a black stroke with `stroke-width="1.5"`. 

Curious to hear your thoughts, @jilaga and @yochannah.
Do you think this is a bit much? Might `1` be better?

Screenshot (705)

**Comment by yochannah** [comment]
yeah, it's a little bit thick now 😂

**Comment by NPDebs** [comment]
I think it looks much better now. What say you, my friends?

**Review by yochannah — APPROVED** [review]
I say yay!

**Pull request merged**